ocf_datapipes/load/nwp/providers/ecmwf.py

```diff
@@ -0,0 +1,36 @@
+"""ECMWF provider loaders"""
+import pandas as pd
+import xarray as xr
+
+from ocf_datapipes.load.nwp.providers.utils import open_zarr_paths
+
+
+def open_ifs(zarr_path) -> xr.DataArray:
+    """
+    Opens the ECMWF IFS NWP data
+
+    Args:
+        zarr_path: Path to the zarr to open
+
+    Returns:
+        Xarray DataArray of the NWP data
+    """
+    # Open the data
+    nwp = open_zarr_paths(zarr_path)
+    # TODO This name should change to be something not UKV
+    ifs: xr.DataArray = nwp["UKV"]
```

Comment:
Just merged [a PR](https://github.com/openclimatefix/nwp-consumer/pull/60) in the consumer to change this, but it varies based on area. Since we know that the xarray dataset always is laid out with a single top level data variable, within which the various parameters are encoded in a "variable" dimension, you could use the generic loading solution:

```python
dataVar: str = list(nwp.data_vars.keys())[0]
```

You could even sanity check it:

```python
dataVars = list(nwp.data_vars.keys())
if  len(dataVars) > 1:
    raise Exception("Too many TLDVs")
else:
    dataVar = dataVars[0]
```

Reply:
Okay, yeah, that would work well!